.travis.yml

```diff
@@ -48,16 +48,26 @@ matrix:
   allow_failures:
     - rust: nightly
   include:
-    - rust: stable
+    - name: Backend
+      rust: stable
       before_install:
-        - nvm install 10
-        - npm ci
         - rustup component add rustfmt-preview
         - rustup component add clippy-preview
       script:
         - cargo fmt -- --check
         - cargo clippy --all-targets --all-features --all
         - cargo test
+    - name: Frontend
+      rust: stable
+      before_install:
+        - nvm install 10
+        - npm ci
```

Comment:
this should probably be moved to the `install` step while we're at it

Reply:
Right. Then we can remove `echo` on install: